01_install_requirements.sh

```diff
@@ -183,3 +183,11 @@ if ! which operator-sdk 2>&1 >/dev/null ; then
     sudo wget https://github.com/operator-framework/operator-sdk/releases/download/v0.9.0/operator-sdk-v0.9.0-x86_64-linux-gnu -O /usr/local/bin/operator-sdk
     sudo chmod 755 /usr/local/bin/operator-sdk
 fi
+
+# Install Go dependency management tool
+# Using pre-compiled binaries instead of installing from source
+if ! which dep 2>&1 >/dev/null ; then
+    mkdir -p $GOPATH/bin
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+    export PATH="${GOPATH}/bin:$PATH"
```

Comment:
For the `which` to work here, I think we have to move the `export PATH` out of the conditional, and also ensure GOPATH is set with eval go env?

Reply:
yeah, sorry for not trying this